packages/html/ds/src/checkbox/checkbox.html

```diff
@@ -0,0 +1,22 @@
+{% from 'checkbox/helpers.html' import getTickSize, getSizeClass, getMarginClass %}
+{% from 'hint-text/hint-text.html' import govieHintText %}
+
+{% macro govieCheckbox(props) %}
+
+<div class="gi-flex gi-flex-col">
+  <div class="gi-flex gi-items-center">
+    <input data-element="{{ props.dataElement }}" id="{{ props.checkboxId }}" value="{{ props.value }}" class="{{ getSizeClass(props.size) | trim }} {{ getTickSize(props.size) | trim }} gi-cursor-pointer gi-appearance-none gi-border-[3px] gi-border-solid gi-border-black checked:before:gi-block checked:before:gi--rotate-45 checked:before:gi-relative checked:before:gi-border-l-[4px] checked:before:gi-border-b-[4px] checked:before:gi-border-black" type="checkbox" />
+    <label for="{{ props.checkboxId }}" class="gi-cursor-pointer gi-font-primary xs:gi-text-sm md:gi-text-md lg:gi-text-lg gi-pl-3.5 ">
+      {{ props.label }}
+    </label>
+  </div>
+  {% if props.hint %}
+    <div class="gi-mb-[-10px]">
```

Comment:
@pregno @hamza14khan 
A small suggestion here. I've had to add the `-10px` value because the `hint-text` has a margin bottom of `10px` already integrated in the component. I think a better way to approach this is to keep the 'hint-text` and `error-text` components without margins since that will be applied from the parent component

Reply:
Or having a props `no-margin` to disable them

Reply:
@hamza14khan is it okay with you if I add the `no-margin` prop for the `hint-text` and `error-text` components

Reply:
Fixed